sql/sqlite/driver.go

```diff
@@ -68,13 +70,25 @@ func Open(db schema.ExecQuerier) (migrate.Driver, error) {
 		return nil, fmt.Errorf("sqlite: scanning database collations: %w", err)
 	}
 	return &Driver{
-		conn:        c,
-		Differ:      &sqlx.Diff{DiffDriver: &diff{c}},
-		Inspector:   &inspect{c},
-		PlanApplier: &planApply{c},
+		conn:               c,
+		Differ:             &sqlx.Diff{DiffDriver: &diff{c}},
+		Inspector:          &inspect{c},
+		PlanApplier:        &planApply{c},
+		RevisionReadWriter: sqlx.NewRevisionStorage(db, dialect.SQLite),
```

Comment:
```suggestion
		RevisionReadWriter: sqlx.NewEntRevisions(db, dialect.SQLite),
```
Maybe `NewEntRevisions` instead?